packages/sdk-components-react-radix/src/select.tsx

```diff
@@ -24,13 +24,11 @@ import {
   ReactSdkContext,
 } from "@webstudio-is/react-sdk/runtime";
 
-// wrap in forwardRef because Root is functional component without ref
-export const Select = forwardRef<
-  HTMLDivElement,
-  ComponentPropsWithoutRef<typeof Root>
->((props, _ref) => {
-  return <Root {...props} />;
-});
+export const Select = forwardRef<HTMLDivElement, ComponentProps<typeof Root>>(
+  ({ value, defaultValue, ...props }) => {
```

Comment:
ref is needed even if not used
select.tsx:27 Warning: forwardRef render functions accept exactly two parameters: props and ref. Did you forget to use the ref parameter?

﻿So please don't delete _args they are for a reason